ui/src/dropdown.tsx

```diff
@@ -170,10 +172,19 @@ const
   getPageSpecification = () => ({ itemCount: PAGE_SIZE, height: ROW_HEIGHT * PAGE_SIZE } as Fluent.IPageSpecification),
   choicesToItems = (choices: Choice[] = [], v?: S | S[]) => choices.map(({ name, label, disabled = false }, idx) =>
     ({ name, text: label || name, idx, checked: Array.isArray(v) ? v.includes(name) : v === name, show: true, disabled })),
-  useItems = (choices?: Choice[], v?: S | S[]) => {
+  useItems = (choices?: Choice[], v?: S | S[], exactSearch?: boolean) => {
     const [items, setItems] = React.useState<DropdownItem[]>(choicesToItems(choices, v))
-    const onSearchChange = (_e?: React.ChangeEvent<HTMLInputElement>, newVal = '') => setItems(items => items.map(i => ({ ...i, show: fuzzysearch(i.text, newVal) })))
 
+    const onSearchChange = (_e?: React.ChangeEvent<HTMLInputElement>, newVal = '') => {
+      setItems((items) =>
+        items.map((i) => ({
+          ...i,
+          show: exactSearch
+            ? exactsearch(i.text, newVal) // Assuming exactsearch is a function for exact matching
+            : fuzzysearch(i.text, newVal), // Assuming fuzzysearch is a function for fuzzy matching
```

Comment:
No need for extra comments.

```suggestion
            ? exactsearch(i.text, newVal)
            : fuzzysearch(i.text, newVal),
```